### PR TITLE
Dimos/new venn plot

### PR DESCRIPTION
**Pull request by dimostzim**
The venn plot has been updated to display unique miRNA sequences instead of families. A new seq_count.py script has been added that works in the same way as family_count.py. 

README files and  png/svg added/updated.

Now family_count.py is necessary for top_families script only, and seq_count.py is necessary for venn.py.

### CODE REVIEW

File: code/plots/Fig2/seq_count.py
Comment (evaklimentova) [inline code/plots/Fig2/seq_count.py:1]: I think that this file is basically the same as `family_count.py`. What about keeping just one script and passing there as an argument name of the column on which w want to do the counting? Would it make sense?

File: code/plots/Fig2/seq_count_README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Description
+
+The `seq_count.py` script processes multiple TSV files containing miRNA sequences data. It counts the occurrences of unique miRNA sequences in each file and generates a summary table. The output is a new TSV file with miRNA sequences as rows and input files as columns, showing the count of each miRNA sequence in each file. This output is necessary for venn.py
```
Comment (evaklimentova) [inline code/plots/Fig2/seq_count_README.md:3]: _input files as columns_ - I think you have fixed column names in the script

File: code/plots/Fig2/seq_count.py
Comment (dimostzim) [reply code/plots/Fig2/seq_count.py:1]: You are right. Merged both of them into one column_count.py.

File: code/plots/Fig2/seq_count_README.md (hunk 1 above)
Comment (dimostzim) [reply code/plots/Fig2/seq_count_README.md]: Updated unified README for merged script

File: code/plots/Fig2/column_count.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    all_counts = []
+    all_values = set()
+    for input_file in input_files:
+        counts = count_column_values(input_file, count_column)
+        all_counts.append(counts)
+        all_values.update(counts.keys())
+    return all_counts, all_values
+
+def get_column_names(input_files, count_column):
+    return [count_column, 'Manakov2022', 'Hejret2023', 'Klimentova2022']
```
Comment (evaklimentova) [inline code/plots/Fig2/column_count.py:32]: I'm still a bit unsure about having these names hardcoded... You then need to be very careful about the parameters when running the script

Comment (dimostzim) [reply code/plots/Fig2/column_count.py]: We know take the file name of each dataset and use it as the corresponding column name in the output

**Pull request merged**